cypress/integration/ete/new_account_review.3.cy.ts

```diff
@@ -86,7 +86,7 @@ describe('New account review page', () => {
 									expect(consent.consented).to.be.true;
 								}
 								if (consent.id === 'personalised_advertising') {
-									expect(consent.consented).to.be.true;
+									expect(consent.consented).to.be.false;
```

Comment:
This was fine as it was.

The problem is on line 66 of the test file, where we click on the advertising checkbox.

With the change in this PR, we don't need that click anymore, as advertising is on by default, and we click to turn it off, whereas before advertising was off by default and we had to click to turn it on.

So we should remove this click, and update the comment

```suggestion
									expect(consent.consented).to.be.true;
```